tests/template_tests/syntax_tests/i18n/test_trans.py

```diff
@@ -0,0 +1,230 @@
+from __future__ import unicode_literals
+
+import os
+from threading import local
+
+from django.conf import settings
+from django.template import Context, Template, TemplateSyntaxError
+from django.test import SimpleTestCase, override_settings
+from django.utils import translation
+from django.utils._os import upath
+from django.utils.safestring import mark_safe
+from django.utils.translation import activate, get_language, trans_real
+
+from ...utils import setup
+
+here = os.path.dirname(os.path.dirname(os.path.abspath(upath(__file__))))
```

Comment:
It would be better to factor out shared elements such as these variables and MultipleLocaleActivationTests setUp/tearDown into a `base.py`.